src/odemis/gui/comp/miccanvas.py

```diff
@@ -1800,6 +1804,7 @@ class FastEMMainCanvas(DblMicroscopeCanvas):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.shapes_overlay = []
```

Comment:
Can you explain what this contains? I think that's the 3 overlays that handle the "lower level" shapes, right?